### ISSUE TITLE
Monitor is inconsistent/not working properly

### ISSUE THREAD

**Issue by psadev** [3proxy/3proxy#827]
Hello,

I've been trying to implement the monitor function in my deployments to add new proxy routes without restarting 3proxy.
This is an example of the current config (in testing environment)

config.cfg
```
auth strong
users test:CL:test123

log /var/log/3proxy/3proxy-%y%m%d.log D
logformat "L%t.%. %N.%p %E %U %C:%c %R:%r %O %I %h %T"
rotate 60

config /etc/3proxy/routes.cfg
monitor /etc/3proxy/routes.cfg
```

routes.cfg
```
proxy -a -p2000 -i1.1.1.1 -e1.1.1.1
```

When I test this feature, I simply edit the routes.cfg file by adding another proxy line (2nd IP on the machine), I then don't touch the file for 30+ secs to avoid locking issues.
After restarting 3proxy and editing routes.cfg once, it seems to work fine, when I try to add/remove another route in the same way it just doesn't work, doesn't matter how much I wait for the reload.

Basically, the monitor feature works for me only once after restarting the 3proxy service, it doesn't work later on, no matter what I do.

Is there something wrong with my usage of the monitor feature or is this a bug?

**Comment by z3APA3A** [comment]
Probably, you mistakenly use `config` instead of `include`.

**Comment by psadev** [comment]
Thank you, that seems to have solved the issue!
